changelogs/unreleased/gh-7200-use-0-as-space-id-for-self-fkey.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
```

Comment:
Commit title is too long. It should be no longer than 50 symbols: https://github.com/tarantool/tarantool/wiki/Code-review-procedure#general-coding-points-to-check